eks-public-cluster.tf

```diff
@@ -11,9 +11,11 @@ module "eks-public" {
   cluster_name = local.public_cluster_name
   # From https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
   cluster_version = var.kubernetes_version
-  subnet_ids      = module.vpc.private_subnets
+  # subnet_ids      = concat(module.vpc.private_subnets, module.vpc.public_subnets)
```

Comment:
```suggestion
```

Reply:
Moved here as it's a value we'll probably want to changr